python/setup.py

```diff
@@ -16,26 +17,77 @@
 from setuptools import setup, find_packages
 from setuptools.command.build_py import build_py
 from setuptools.command.develop import develop
+from setuptools.command.install import install
 from setuptools.command.test import test as test_command
 from typing import List
+from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
 
 PLATFORM = 'unix'
 if platform.platform().startswith('Win'):
     PLATFORM = 'win'
 
 MODEL_DIR = os.path.join('stan', PLATFORM)
 MODEL_TARGET_DIR = os.path.join('prophet', 'stan_model')
-
+CMDSTAN_VERSION = "2.26.1"
 
 def get_backends_from_env() -> List[str]:
-    from prophet.models import StanBackendEnum
```

Comment:
This was causing the build system to fail, since the build happens in an isolated environment with only the dependencies needed for `setup.py`. I think removing this makes things a bit cleaner, but keen to hear thoughts.